crates/subspace-networking/src/node.rs

```diff
@@ -368,6 +406,53 @@ impl Node {
         }
     }
 
+    /// Announce piece by its key. Initiate 'start_providing' Kademlia operation.
+    pub async fn announce_piece(&self, key: Multihash) -> Result<(), AnnouncePieceError> {
```

Comment:
Should this be just `announce` instead? I'm not sure network actually cares if it is a piece or something else entirely.